snapcraft/tests/plugins/test_meson.py

```diff
@@ -0,0 +1,118 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from unittest import mock
+from testtools.matchers import HasLength
+
+import snapcraft
+from snapcraft import tests
+from snapcraft.plugins import meson
+
+
+class MesonPluginTestCase(tests.TestCase):
+    """Plugin to provide snapcraft support for the meson build system"""
+
+    def setUp(self):
+        super(MesonPluginTestCase, self).setUp()
```

Comment:
No need to define the setUp if it just calls the parent setUp. That's the default action.